content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -1,48 +1,52 @@
-"Alpine Linux","Alpine Linux is a security-oriented, lightweight Linux distribution based on musl libc and busybox. "
-"Anaconda","Anaconda is the installation program used by Fedora, RHEL, and others. POWER support will be used for Continuous Integration and/or development of Project Anaconda."
-"Apache Bigtop","Bigtop is an Apache Foundation project for Infrastructure Engineers and Data Scientists looking for comprehensive packaging, testing, and configuration of the leading open source big data components. Bigtop supports a wide range of components/projects, including, but not limited to, Hadoop, HBase and Spark."
-"Apache Software Foundation","The ASF is currently working on supporting POWER builds for CouchDB and possibly other ASF projects in the future. The current goal is to support Cloudant on POWER."
-"BearSSL","BearSSL is a SSL/TLS library, with a focus on having a clean, secure implementation of the protocol, up-to-date with the latest research (constant-time crypto algorithms and processing). System dependencies are reduced to the bare minimum (no dynamic memory allocation even) to help use on constrained, embedded systems with little or no operating system. Access to POWER8 systems would allow me to write some specific implementations of AES/GCM and other cryptographic algorithms that leverage the specific opcodes offered by the CPU, for better performance. "
-"BLCR","Berkeley Lab Checkpoint/Restart (BLCR) for LINUX."
+"Anaconda","Installation program used by Fedora, RHEL, and others; uses POWER support for Continuous Integration and development"
+"Apache Bigtop","An Apache Foundation project for Infrastructure Engineers and Data Scientists looking for comprehensive packaging, testing, and configuration of the leading open source big data components; supports a wide range of components/projects, including, but not limited to, Hadoop, HBase and Spark"
+"Apache Software Foundation","Supports POWER builds for CouchDB and possible other ASF projects in the future; currently working to support Cloudant on POWER"
+"BearSSL","An SSL/TLS library, with a focus on a clean, secure implementation of protocol, up-to-date with the latest research; POWER8 systems writes specific implementations of AES/GCM and other cryptographic algorithms that leverage the specific opcodes offered by the CPU, for better performance"
+"`BLCR`_","Berkeley Lab Checkpoint/Restart for LINUX is a kernel-level checkpointer implemented via a loadable kernel module and a userspace library. They use the system to compile, test and debug kernel modules against a range of linux kernels."
```

Comment:
I mentioned that to @knightsamar and he said he wanted to showcase the difference between the project and the partner. Should I instead move this description (since it's more thorough) to the academic partner and get rid of this?